static/app/views/settings/account/notifications/utils.tsx

```diff
@@ -41,22 +41,21 @@ export function getDocsLinkForEventType(
   event: DataCategoryExact | string // TODO(isabella): get rid of strings after removing need for backward compatibility on gs
 ) {
   switch (event) {
-    case DataCategoryExact.TRANSACTION || 'transaction':
```

Comment:
This should resolve to be `DataCategoryExact.TRANSACTION || 'transaction' === DataCategoryExact.TRANSACTION` therefore the right hand side of the `||` isn't needed.